warp10/src/main/java/io/warp10/standalone/StandaloneIngressHandler.java

```diff
@@ -357,20 +361,30 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
         
         if (dr.getAttributesSize() > 0) {
           if (null != dr.getAttributes().get(StandaloneAcceleratedStoreClient.ATTR_NOCACHE)) {
-            nocache = true;
-          } else {
-            nocache = false;
+            forcedNocache = true;
+            nocache = !"false".equals(dr.getAttributes().get(StandaloneAcceleratedStoreClient.ATTR_NOCACHE));
```

Comment:
In DELETE, FETCH and UPDATE, the test is `Boolean.TRUE.equals` so for coherency it should be `"true".equals`.

Reply:
The reason why we are checking if the attribute is "false" is because prior to this PR the attribute was simply set to the empty string to specify that it was true, not to 'true', so in order to remain backward compatible with DatalogRequests which might still be buffered, we check if the attribute is "false" and negate the test.